apollo-router/src/uplink/mod.rs

```diff
@@ -123,6 +126,13 @@ impl Endpoints {
             }
         }
     }
+
+    pub fn url_count(&self) -> usize {
```

Comment:
Does it really need to be pub instead of just `pub(crate)` ?

Reply:
Thanks - I wasn't aware of the differences but I just looked it up and it definitely should be pub(crate)